Formula/libstfl.rb

```diff
@@ -71,16 +73,13 @@ def install
       end
     end
 
+    inreplace "perl5/Makefile.snippet", "perl Makefile.PL", "perl Makefile.PL INSTALL_BASE=#{libexec}"
+
     # Fails race condition of test:
     #   ImportError: dynamic module does not define init function (init_stfl)
     #   make: *** [python/_stfl.so] Error 1
     ENV.deparallelize
-
     system "make"
-
-    on_macos do
-      inreplace "perl5/Makefile", "Network/Library", libexec/"lib/perl5"
-    end
```

Comment:
Probably need to test this.
On macOS, this will change the final directory structure.

Before:
```
/usr/local/opt/libstfl/libexec/lib/perl5/
└── Perl
    └── 5.28
        └── darwin-thread-multi-2level
            ├── auto
            │   └── stfl
            │       └── stfl.bundle
            ├── example.pl
            ├── example2.pl
            └── stfl.pm
```

After:
```
/usr/local/opt/libstfl/libexec/lib/perl5/
└── darwin-thread-multi-2level
    ├── auto
    │   └── stfl
    │       └── stfl.bundle
    ├── example.pl
    ├── example2.pl
    ├── perllocal.pod
    └── stfl.pm
```